app/views/practices/form/metrics.html.erb

```diff
@@ -0,0 +1,274 @@
+              <thead>
+                <tr>
+                  <th scope="col" class="width-80">Phase</th>
+                  <th scope="col">Adoptions</th>
+                </tr>
+              </thead>
+              <tr>
+                <td class="width-80">Completed</td>
+                <td><%= @adoptions_successful_total_at %></td>
+              </tr>
+              <tr>
+                <td class="width-80">In-progress</td>
+                <td><%= @adoptions_in_progress_total_at %></td>
+              </tr>
+              <tr>
+                <td class="width-80">Unsuccessful</td>
+                <td><%= @adoptions_unsuccessful_total_at %></td>
+              </tr>
+              <tr>
+                <td class="width-80">Total</td>
+                <td><%= @adoptions_total_at %></td>
+              </tr>
+            </table>
+            <table class="usa-table usa-table--borderless width-full margin-bottom-5 metrics-table">
+              <thead>
+                <tr>
+                  <th scope="col" class="width-80">Facility type</th>
+                  <th scope="col">Adoptions</th>
+                </tr>
+              </thead>
+              <tr>
+                <td class="width-80">Rural</td>
+                <td><%= @rural_facility_at %></td>
+              </tr>
+              <tr>
+                <td class="width-80">Urban</td>
+                <td><%= @urban_facility_at %></td>
+              </tr>
+            </table>
+            <table class="usa-table usa-table--borderless width-full margin-bottom-5 metrics-table">
+              <thead>
+                <tr>
+                  <th scope="col" class="width-80">Facility complexity</th>
+                  <th scope="col">Adoptions</th>
+                </tr>
+              </thead>
+              <tr>
+                <td class="width-80">1a (High Complexity)</td>
+                <td><%= @a_high_complexity_at %></td>
+              </tr>
+              <tr>
+                <td class="width-80">1b (High Complexity)</td>
+                <td><%= @b_high_complexity_at %></td>
+              </tr>
+              <tr>
+                <td class="width-80">1c (High Complexity)</td>
+                <td><%= @c_high_complexity_at %></td>
+              </tr>
+              <tr>
+                <td class="width-80">2 (Medium Complexity)</td>
+                <td><%= @medium_complexity_2_at %></td>
+              </tr>
+              <tr>
+                <td class="width-80">3 (Low Complexity)</td>
+                <td><%= @low_complexity_3_at %></td>
+              </tr>
+            </table>
+          </div>
+       </div>
+
+<!-- LEADER BOARD -------------------------------------------------------------------------------------------->
+        <div class="grid-row flex-align-center margin-bottom-3">
+          <h2 class="grid-col-6 line-height-36 font-sans-xl margin-y-0">Leader board</h2>
```

Comment:
no space in between leader and board

Reply:
Leaderboard vs Leader board...  :) done.